src/jobs/database_schema_report.yml

```diff
@@ -21,8 +25,8 @@ parameters:
     default: "localhost"
     description: Host name of the database
   port:
-    type: string
-    default: "5432"
+    type: integer
+    default: 5432
```

Comment:
This is technically a breaking change but no-one is specifying it. (and if they were the job would be broken)